### DISCUSSION TITLE
Adding an --arch option?

### DISCUSSION THREAD

**Discussion by michael-picard** [dbrekelmans/browser-driver-installer#100] [General]
Hello there,

May I suggest adding an --arch option ? 

The idea is to be able to choose the architecture when installing a driver like this:

`vendor/bin/bdi driver:chromedriver --os=linux --arch=linux-arm64`

thanks,
Michael

**Comment by dbrekelmans** [comment]
Hi @michael-picard,

I've had plans in the past to support ARM drivers (https://github.com/dbrekelmans/browser-driver-installer/issues/67), but never got around to it.

I like the idea of an `--arch` flag. We'd need to check of course of the combination of OS and architecture is available as a driver. In addition, the detect command should automatically detect the architecture and download to corresponding driver.

If you'd like to give it a try, I'll be more than happy to review your PR!

**Comment by michael-picard** [comment]
Hi @dbrekelmans , thanks for your reply. I will give this a try in the coming days.

**Comment by michael-picard** [comment]
Hi @dbrekelmans, I’ve created the PR for your review. I’ve spent around 4 hours working on it so far, which is the time my company allocated for this task. Let me know if there’s anything that needs adjustment or improvement.
